README.md

```diff
@@ -185,4 +189,34 @@ To deploy the `dynatrace-aws-s3-log-forwarder` using the provided container imag
 
 ### Next steps
 
-At this stage, you should see logs being ingested in Dynatrace as they're written to Amazon S3. For more detailed information and advanced configuration details, visit the documentation in the `docs` folder.
+At this stage, you should see logs being ingested in Dynatrace as they're written to Amazon S3.
+
+You can explore logs using the Dynatrace [Logs and events viewer](https://docs.dynatrace.com/docs/observe-and-explore/logs/log-management-and-analytics/lma-analysis/logs-and-events). You can also create metrics and alerts based on ingested logs (see [Log metrics](https://docs.dynatrace.com/docs/observe-and-explore/logs/log-management-and-analytics/lma-analysis/lma-log-metrics) and [Log events](https://docs.dynatrace.com/docs/observe-and-explore/logs/log-management-and-analytics/lma-analysis/lma-log-events) documentation).
+
+You can also perform deep log analysis with [Dynatrace Notebooks](https://docs.dynatrace.com/docs/observe-and-explore/notebook) and [Dynatrace Query Langage (DQL)]((https://docs.dynatrace.com/docs/platform/grail/dynatrace-query-language/dql-guide)). See some example DQL queries below:
```

Comment:
We have a link to DQL documentation at the bottom, so maybe we could remove second part of this sentence.

Reply:
Updated wording